Yoast/wordpress-seo
# The class isn't used anymore

## Context


* Removal of a class that isn't used at the moment.

## Summary


This PR can be summarized in the following changelog entry:

* There is no implementation of it. 

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* This pull removes a class that isn't used anymore. Make sure the build passes. A search for the class name learns there is no real use.

## UI changes

## Documentation

## Quality assurance

* [x] I have ~added~removed unittests to verify the code works as intended

Fixes #
